.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r requirements-dev.txt
```

Comment:
I recommend putting separate commands into separate steps because they are timed separately and displayed better in the log this way. When something fails, it's easier to narrow down what went wrong too.